docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -111,7 +112,7 @@ Following are the available properties for each layer type:
 
```

Comment:
@daeinki 
How about this?


```suggestion
Following are the available properties for each layer type:
> [!NOTE]
> All the properties listed in the following table are universal and apply to each layer. 

Type | Key | value | Description
---------- | --- | ----- | -----------
&#xfeff;                    | name               | (string)      | An identifier for each layer
.....

```

Let me know your opinion, It is not wrong currently, but this would be a more formal way of writing. 

If you are fine with this suggestion, I would proceed with commenting on other places (where ever applicable).

Reply:
Do you also have idea about to make it more brief? 

My concern is to keep the property list tidy considering that the properties would be added version by version.

Currently..
```
1. layer
@6.0
   (table)
@6.5
   (table)
@7.0...
   (table)
1. optimizer
   (omitted)
```

If I understood correct, the suggestion is,
```
1. layer
@6.0
   (universal table)
   (type-dependent table)
@6.5
   (universal table)
   (type-dependent table)
@7.0...
   (universal table)
   (type-dependent table)
1. optimizer
   (omitted)
```

Reply:
What my idea was to have the text "universal properties" in the leading line before the table.

Reply:
Sorry that I might have not clarified the table enough.

IMO, description `The following are the available universal properties that apply to each layer type:`  would be a bit misleading without separating the table. 

Please confer to the below image from the [staged doc](https://docs1.stg.tizen.org/staging/1418/application/native/guides/machine-learning/machine-learning-training/)

![image](https://user-images.githubusercontent.com/17230883/113811565-697c0480-97a7-11eb-8b52-2d7be49f1c26.png)

Reply:
@zhoonit Thanks, yes then the leading sentence is inappropriate. Thanks for the clarity.